src/loc.rs

```diff
@@ -28,7 +28,7 @@ impl LocDiff {
     /// The getter method for the field `file` of the corresponding struct.
     ///
     /// Since the struct `LocDiff` defines its field `file` to be private, this
-    /// getter is utilised in order to operate on this field.  It returns a
+    /// getter is used in order to operate on this field.  It returns a
```

Comment:
Why did you rephrase here from utilised to used? Especiially, since above you actually wrote utilised again?

Reply:
Above, I forgot to change it.  And rephrasing is a meaningful change since utilise does not express exactly what it should have had to.